packages/google_sign_in/android/src/main/java/io/flutter/plugins/googlesignin/GoogleSignInPlugin.java

```diff
@@ -398,25 +408,24 @@ public boolean onActivityResult(int requestCode, int resultCode, Intent data) {
 
   private void onSignInResult(GoogleSignInResult result) {
     if (result.isSuccess()) {
-      finishWithSuccess(getSignInResponse(result.getSignInAccount()));
-    } else {
+      GoogleSignInAccount account = result.getSignInAccount();
+      Map<String, Object> response = new HashMap<>();
+      response.put("displayName", account.getDisplayName());
```

Comment:
According to documentation [here](https://developers.google.com/android/reference/com/google/android/gms/auth/api/signin/GoogleSignInAccount.html#getDisplayName()): This is not guaranteed to be present for all users.

We should do a null check for display name just like we are doing for photoUrl below. If it is null, let's set it to empty string?